user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -79,33 +80,56 @@ The elements will be created with the following specifications:
 - **CCAP:**
 
   - **Element name**: The defined element name.
+
   - **Protocol**: The selected protocol.
+
   - **Protocol version**: The selected version.
+
   - **IP address**: The defined IP address.
+
   - **Alarm template**: The default alarm template.
+
   - **Trend template**: The default trend template.
+
   - **Get community string CCAP**: The defined get community string for the CCAP.
+
   - **Set community string CCAP**: The defined set community string for the CCAP.
+
   - **Entity export directory**: The same directory as is set in the back end of the host.
+
   - **Entity import directory**: The same directory as is set in the back end of the host.
+
   - **Entity import directory type**: Remote.
-  - **System username**: The system username.
-  - **System password**: The system password.
+
+  - **System username**: The username to access the shared folder.
+
+  - **System password**: The password to access the shared folder.
 
 - **Collector:**
 
   - **Element name**: The defined element name combined with the suffix "_COLLECTOR".
-  - **Protocol** The [Generic DOCSIS CM Collector](https://catalog.dataminer.services/result/driver/4207) protocol.
+
+  - **Protocol** The [Generic DOCSIS CM Collector](https://catalog.dataminer.services/details/connector/4207) protocol.
+
   - **Protocol version**:The production version.
+
   - **IP address**: 127.0.0.1.
+
   - **Alarm template**: public.
+
   - **Trend template**: private.
+
   - **Get community string Collector**: The defined get community string for the collector.
+
   - **Set community string Collector**: The defined set community string for the collector.
+
   - **Entity import directory**: The same directory as is set in the back end of the host.
+
   - **Entity import directory type**: Remote.
-  - **System username**: The system username.
-  - **System password**: The system password.
+
+  - **System username**: The username to access the shared folder.
+
+  - **System password**: The password to access the shared folder.
 
 > [!NOTE]
 > It may take some time before the elements are created. The larger the cluster, the longer it will take to create the elements.
```

Comment:
@EstebanSkyline, regarding the name of the DMA, can a user set the DMA ID instead of the DMA name?
If so, can we include a note that indicates that a user can use either DMA ID or DMA name?
else, can we include a note that indicates that currently we only support DMA name?

Reply:
@miguel-slc, I've now added a note with this info. Could you confirm if the PR is OK to merge now?

Reply:
Hi @MariekeGO, yes, it is ok to merge